modules/35-blackbox-rightclick.yml

```diff
@@ -0,0 +1,10 @@
+name: blackbox-rightclick
+type: shell
+commands:
+  - mkdir -p /usr/share/nautilus-python/extensions/
+  - cp /sources/blackbox-rightclick/nautilus-open-in-blackbox-vos-1/nautilus-open-in-blackbox.py /usr/share/nautilus-python/extensions/
+
+source:
+  type: zip
```

Comment:
vib doesn't have a zip type, you'd have to use a tar archive here

Reply:
Ah alr